azavea/raster-vision-aws
# Allow any instance from r5 family by default

Overview
-----

This PR allows the CPU environment to use any instance type from the r5 family instead of only r5.larges.

Because the job runner submits many copies of the same job with a single vCPU required, allowing the spot fleet to mix and match instances based on total resource needs instead of needing 16 copies of a specific instance type should in principal lead to getting instances sooner. Also, because spot market prices can sometimes get out of line with on-demand price ratios, this should allow the spot fleet to save more money by choosing instance types to meet capacity at the lowest price per resource unit.

Testing
-----

- Create a stack from this template
- Make sure it succeeds
- Try to run some jobs in the CPU compute environment based on that stack
- confirm that the spot request can be fulfilled by several instance types
